src/reducers/starWarsReducer.js

```diff
@@ -1,13 +1,32 @@
-import /* we need our action types here*/ "../actions";
+import {FETCH_START,FETCH_SUCCESS,FETCH_FAIL} from '../actions';
+
 const initialState = {
-  characters: []
+  characters: [],
+  fetching: false,
```

Comment:
as of right now you're not utilizing a way to handle an incoming error from the API: set a base prop `error: null` here and update/reset it as needed within your cases below, and use the payload from the error case to set it on state here, so your app can read if there is an error and show the user some text to that effect. look at Dustin's demo from yesterday (Wednesday) in more depth to get an idea of what I mean